cardano-db/src/Cardano/Db/Schema.hs

```diff
@@ -304,14 +300,23 @@ share
     addrId              StakeAddressId      noreference
     type                RewardSource        sqltype=rewardtype
     amount              DbLovelace          sqltype=lovelace
-    earnedEpoch         Word64
+    earnedEpoch         Word64 generated="((CASE WHEN (type='refund') then spendable_epoch else (CASE WHEN spendable_epoch >= 2 then spendable_epoch-2 else 0 end) end) STORED)"
```

Comment:
is there no way to do this functionality (and bellow) in the codebase? Just incase this functionality gets lost here, failing that a comment as to why it's here and what it does would suffice.

Reply:
This functionality exists in Haskell code as well, but is ignored, due to the generated field.